gateway/xgress_edge_wss/dialer.go

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
I don't think this xgress component needs to be able to dial. I think this whole thing can be blanked out.

Reply:
deleted.  new wss transport is in place.